index.Rmd

```diff
@@ -0,0 +1,461 @@
+  geom_hline( aes(yintercept = 60)) +
+  geom_hline( aes(yintercept = 50) , 
+              color = "orange") +
+  theme_gray() +
+  labs(
+        y = "Two Day Change in Degrees" ,
+    title = "Two Day Temperature Change in Degrees F" ,
+    subtitle = "Temperature Variation 2019 - 2021",
+    color = "Weather Station Location",
+    caption = "Data from www.noaa.gov")
+
+
+weather %>% 
+  ggplot(aes( x = DATE,
+              y =Temp_1Week_diff, 
+              color = Location)) +
+  geom_point(na.rm = TRUE) +
+  geom_hline( aes(yintercept = 60)) +
+  geom_hline( aes(yintercept = 50),
+              color = "orange") +
+  theme_gray() +
+    labs(
+        y = "One Week Change in Degrees" ,
+    title = "One Week Temperature Change in Degrees F" ,
+    subtitle = "Temperature Variation 2019 - 2021",
+    color = "Weather Station Location",
+    caption = "Data from www.noaa.gov") 
+```
+ 
+ 
+
+South Dakota and Oklahoma weather stations had more of the weather trends where the high and low over several days were far enough apart to cause extra battery stress. The other 2 stations did have a few such trends, but based on this SD and OK are places to check into futher and prepare for more battery wear. 
+ 
+ 
+# Number of Extreme Temp Days
+
+## Per Station, Per Year
+```{r, warning=FALSE, message = FALSE  }
+weather %>% 
+  count(cold_flag, hot_flag) 
+  
+  
+weather %>% 
+    group_by(Location, Year) %>% 
+  summarise( num_hot_days = sum(as.numeric(hot_flag)),
+             num_cold_days = sum(as.numeric(cold_flag))) %>% 
+  ungroup() %>% 
+  ggplot( aes ( x =Year ,
+                 y=  num_hot_days,
+                color = Location,
+                fill = Location)) +
+  geom_col(position = "dodge") +
+  theme_bw() +
+  theme(legend.position = "bottom" ,
+        legend.direction = "vertical") +
+      labs(
+        y = "Number of Extreme Days" ,
+    title = "Number of Extreme Hot Days Per Year, by Location" ,
+    caption = "Data from www.noaa.gov")
+
+
+```
+
+
+Based on the previous graphs, the extreme hot flag and extreme cold flag are not expected on the same day.  But when looking at the number of extreme hot days, OK has the most.  The PA and OH have the next highest count for hot days.  More data will be needed to determine if those 2 areas should be treated the same, but the OK area is clearly different. 
+
+
+
+```{r,  warning=FALSE, message = FALSE  }
+weather %>% 
+  group_by(Location, Year) %>% 
+  summarise( num_hot_days = sum(as.numeric(hot_flag)),
+             num_cold_days = sum(as.numeric(cold_flag))) %>% 
+  ungroup() %>% 
+  ggplot( aes ( x =Year ,
+                 y=  num_cold_days,
+                color = Location,
+                fill = Location)) +
+  geom_col(position = "dodge") +
+  theme_bw() +
+  theme(legend.position = "bottom" ,
+        legend.direction = "vertical") +
+      labs(
+        y = "Number of Extreme Days" ,
+    title = "Number of Extreme Hot Days Per Year, by Location" ,
```

Comment:
I believe that you meant to change the title here to "Number of Extreme Cold Days Per Year, by Location"